lib/auto-languageclient.js

```diff
@@ -81,7 +81,22 @@ export default class AutoLanguageClient {
   getInitializeParams(projectPath: string, process: child_process$ChildProcess): ls.InitializeParams {
     return {
       processId: process.pid,
-      capabilities: {},
+      capabilities: {
+        workspace: {
+          applyEdit: true,
+          workspaceEdit: {
+            documentChanges: true
+          }
+        },
+        textDocument: {
+          synchronization: {
+            dynamicRegistration: false,
+            willSave: true,
+            willSaveWaitUntil: false,
+            didSave: true
```

Comment:
Instead of sending over only what wouldn't be default values, should I be explicit and send the whole block even if most of them will have false values?

Reply:
I don't have a strong preference here - initialize is only called once so no perf cost to send them all.  I doubt they'd change default either.

Reply:
Small +1 to sending them all, as this would also be a good quick bit of documentation for what's implemented so far, and may also serve as a reminder to send up a `true` once a feature is implemented.

Reply:
Yep, planning to add the rest of them before merging!